source/Octopus.Tentacle.Client/Scripts/ScriptOrchestratorFactory.cs

```diff
@@ -63,9 +63,9 @@ public async Task<IScriptOrchestrator> CreateOrchestrator(CancellationToken canc
             {
                 scriptServiceToUse = await DetermineScriptServiceVersionToUse(cancellationToken).ConfigureAwait(false);
             }
-            catch (OperationCanceledException) when (cancellationToken.IsCancellationRequested)
+            catch (Exception ex) when (cancellationToken.IsCancellationRequested)
```

Comment:
This fix is being cherry-picked from https://github.com/OctopusDeploy/OctopusTentacle/pull/730/files?w=1#diff-01651ec96a501b2d869c5c6b153ad18c8c515516dcec657d54a55a6e7e80f5feR67 as it is causing a lot of build failures